notebooks/Double Machine Learning Examples.ipynb

```diff
@@ -0,0 +1,965 @@
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlsAAAF3CAYAAACSQ46hAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAIABJREFUeJzs3Xlc1VX+x/HXuXBZFBAEcUNcQQURRMQ9d7O0xSXNylGbSnMsy2nKaaqpppos1xRL07RcsrIsa8pAyyV3JBcUASFU3FBQQHbuPb8/LH6Wyea9XMDP8/Hoofd+v99zPizB2/M933OU1hohhBBCCGEdBlsXIIQQQghRm0nYEkIIIYSwIglbQgghhBBWJGFLCCGEEMKKJGwJIYQQQliRhC0hhBBCCCuSsCWEEEIIYUUStoQQQgghrEjClhBCCCGEFUnYEkIIIYSwIntbF3AtLy8v3aJFC1uXIYQQQghRpv3791/UWjco67xqFbZatGhBdHS0rcsQQgghhCiTUupEec6T24hCCCGEEFYkYUsIIYQQwookbAkhhBBCWFG1mrP1Z4qKikhNTSU/P9/WpYgKcHJywsfHB6PRaOtShBBCCJuq9mErNTUVV1dXWrRogVLK1uWIctBak56eTmpqKi1btrR1OUIIIYRNVfvbiPn5+Xh6ekrQqkGUUnh6espopBBCCEENCFuABK0aSL5mQgghxFU1ImzZmp2dHSEhIQQGBhIcHMycOXMwm80AbNmyBaUUy5YtKzn/559/RinFrFmzAJgwYQLr1q2zWD0rVqzgzJkzJa8feeQRjh49etPtpqSksGbNmnKdO2TIENzd3Rk2bNhN9yuEEELUZhK2ysHZ2ZkDBw5w5MgRoqKi+Pbbb3nllVdKjgcFBfHJJ5+UvF67di3BwcFWq+ePYWvp0qUEBATcdLsVCVv/+Mc/WLly5U33KYQQQtR2ErYqyNvbmyVLlrBw4UK01gD4+vqSn5/P+fPn0VqzceNG7rjjjgq1u2rVKsLDwwkJCWHSpEmYTCZMJhMTJkygQ4cOBAUFMXfuXNatW0d0dDQPPvggISEh5OXl0bdv35KV911cXHjuuefo3LkzAwcOZO/evfTt25dWrVqxYcMG4Gqo6t27N6GhoYSGhrJz504AZsyYwfbt2wkJCWHu3LmYTCb+8Y9/0KVLFzp27MjixYtL6h0wYACurq6W+JQKIYQQtVq1fxrxWq98fYSjZ7Is2mZAEzf+fVdgha5p1aoVZrOZtLS0kvdGjRrFZ599RqdOnQgNDcXR0bHc7cXFxfHJJ5+wY8cOjEYjU6ZMYfXq1QQGBnL69GliY2MBuHz5Mu7u7ixcuJBZs2YRFhZ2XVs5OTn07duXmTNnMnz4cF544QWioqI4evQo48eP5+6778bb25uoqCicnJxITExk7NixREdH8+abbzJr1iy++eYbAJYsWUK9evXYt28fBQUF9OzZk8GDB8sThkIIIUQF1KiwVZ38Nqr1m9GjRzNmzBiOHTvG2LFjS0aLymPz5s3s37+fLl26AJCXl4e3tzd33XUXycnJPPHEEwwdOpTBgweX2ZaDgwNDhgwBrt7edHR0xGg0EhQUREpKCnB17bKpU6dy4MAB7OzsSEhI+NO2IiMjOXToUMl8s8zMTBITEyVsCSGEqLa01tXuIa0aFbYqOgJlLcnJydjZ2eHt7U1cXBwAjRo1wmg0EhUVxfz58ysUtrTWjB8/nv/+97/XHTt48CDff/89ERERfPrpp3zwwQeltmU0Gku+yQwGQ8kIm8FgoLi4GIC5c+fSsGFDDh48iNlsxsnJ6YZ1LViwgNtvv73cH4sQQghhKzlFOfxt8994uMPD3OZzm63LKSFztirowoULTJ48malTp16XnF999VVmzpyJnZ1dhdocMGAA69atK7ktmZGRwYkTJ7h48SJms5mRI0fyn//8h5iYGABcXV3Jzs6u9MeQmZlJ48aNMRgMrFy5EpPJ9Kft3n777bz77rsUFRUBkJCQQE5OTqX7FUIIIaxpeexyDqQdsHUZ16lRI1u2kpeXR0hICEVFRdjb2zNu3DimT59+3Xk9evS4YRuTJk3iqaeeAqBZs2bs2rWr5FhAQACvvfYagwcPxmw2YzQaiYiIwNnZmYkTJ5YsM/HbyNeECROYPHkyzs7Ov2unvKZMmcLIkSP57LPP6NevH3Xr1gWgY8eO2NvbExwczIQJE5g2bRopKSmEhoaitaZBgwZ8+eWXAPTu3Ztjx45x5coVfHx8WLZsmYyACSGEsKlJHSfRrXE3whpdP6fZltQf5x7ZUlhYmP7tqbrfxMXF0b59extVJG6GfO2EEEJYm9aaD2I/YLjfcOo71a/SvpVS+7XWZSY7uY0ohBBCiBprwc8LmBczj2+SvrF1KTckYUsIIYQQNdJHRz7i/cPvM8p/FOMCxtm6nBuyWthSSrVVSh245r8spdRT1upPCCGEELeO9YnreTv6bQY1H8QLXV+odss9XMtqE+S11vFACIBSyg44Day3Vn9CCCGEuDUUmYpYfmQ5PZv05M3eb2JnqNgqAFWtqp5GHAAkaa1PVFF/QgghhKiljHZGPrj9A+rY18HBzsHW5ZSpquZs3Q98XEV9CSGEEKIWijkfw8s7X6bIXISXsxd1jHVsXVK5WD1sKaUcgLuBz25w/DGlVLRSKvrChQvWLqdS7OzsCAkJITAwkODgYObMmVOy9tWWLVtQSrFs2bKS83/++WeUUsyaNQu4ui7Wb1veWMKKFSs4c+ZMyetHHnmEo0eP3nS7KSkprFmzpszzDhw4QPfu3QkMDKRjx4588sknN923EEIIUZq49Dj+tvlv7D+/n5zC
```

Comment:
Can you un-scale the income? It makes sense to keep the notebooks consistent.

Reply:
I actually liked the idea of scaled income. I’m thinking we should scale it in the other notebooks for consistency.

Reply:
I just think that it makes the plots the harder to interpret, especially if someone wants to compare this to their knowledge of how this curve is supposed to look like.

Reply:
Do we even need to scale W? Per my understanding, only distance based model need to be scaled before training? 

Another finding for the ORF notebook, the x-axis label should be "Income" instead of "log(Income)"